install/bootstrap_livecd_remote_iso.md

```diff
@@ -161,128 +220,120 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
+   volume groups and raid arrays that may be using the disk. **These commands only need to be run if the earlier
+   `parted` command failed.**
 
-   1. Set helper variables
+   ```bash
+   pit# RAIDS=$(grep "${disk}[0-9]" /proc/mdstat | awk '{ print "/dev/"$1 }') ; echo $RAIDS
+   pit# VGS=$(echo $RAIDS | xargs -r pvs --noheadings -o vg_name 2>/dev/null) ; echo $VGS
+   pit# echo $VGS | xargs -r -t -n 1 vgremove -f -v
+   pit# echo $RAIDS | xargs -r -t -n 1 mdadm -S -f -v
+   ```
+
+   After running the above procedure, retry the `parted` command which failed. If it succeeds, resume the install from that point.
+
+1. Mount local disk.
+
+   > **Note:** The FSLabel `PITDATA` is already in `/etc/fstab`, so the path is omitted in the following call to `mount`.
+
+   ```bash
+   pit# mount -vL PITDATA &&
+        mkdir -pv $PITDATA/{admin,configs} $PITDATA/prep/{admin,logs} $PITDATA/data/{k8s,ceph}
+   ```
+
+1. Relocate the typescript to the newly mounted `PITDATA` directory.
+
+   1. Quit the typescript session with the `exit` command.
+
+   1. Copy the typescript file to its new location.
 
       ```bash
-      pit# ENDPOINT=https://arti.dev.cray.com/artifactory/shasta-distribution-stable-local/csm
-      pit# export CSM_RELEASE=csm-x.y.z
-      pit# export SYSTEM_NAME=eniac
+      pit# cp -v ~/csm-install-remoteiso.*.txt $PITDATA/prep/admin
       ```
 
-   1. Save the `CSM_RELEASE` for usage later; all subsequent shell sessions will have this variable set.
+   1. Restart the typescript, appending to the previous file.
+
+      ```bash
+      pit# script -af $(ls -tr $PITDATA/prep/admin/csm-install-remoteiso*.txt | head -n 1)
+      pit# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+      ```
+
+1. Download the CSM software release to the PIT node.
 
-      > The `echo` prepends a newline to ensure that the variable assignment occurs on a unique line,
-      > and not at the end of another.
+   1. Set variable to URL of CSM tarball.
 
-      ```bash      
-      pit# echo -e "\nCSM_RELEASE=$CSM_RELEASE" >>/etc/environment
+      ```bash
+      pit# URL=https://arti.dev.cray.com/artifactory/shasta-distribution-stable-local/csm/${CSM_RELEASE}.tar.gz
       ```
 
    1. Fetch the release tarball.
 
       ```bash
-      pit# wget ${ENDPOINT}/${CSM_RELEASE}.tar.gz -O /var/www/ephemeral/${CSM_RELEASE}.tar.gz
+      pit# wget $URL -O ${CSM_PATH}.tar.gz
       ```
 
    1. Expand the tarball on the PIT node.
 
-      > Note: Expansion of the tarball may take more than 45 minutes.
+      > **Note:** Expansion of the tarball may take more than 45 minutes.
 
       ```bash
-      pit# tar -C /var/www/ephemeral -zxvf /var/www/ephemeral/${CSM_RELEASE}.tar.gz
-      pit# CSM_PATH=/var/www/ephemeral/${CSM_RELEASE}
-      pit# echo $CSM_PATH
-      pit# echo -e "\nCSM_PATH=$CSM_PATH" >>/etc/environment
-      pit# ls -l ${CSM_PATH}
+      pit# tar -C $PITDATA -zxvf ${CSM_PATH}.tar.gz && ls -l ${CSM_PATH}
```

Comment:
{} around PITDATA for consistency.

Reply:
And throughout the page.

Reply:
Got it. In addressing this, I found some prexisting cases of that inconsistency as well, and corrected them.